src/App.tsx

```diff
@@ -11,8 +13,10 @@ import "./styles.css";
  */
 export const App: React.FC = () => {
   return (
-    <Auth.Provider>
-      <Navigation />
-    </Auth.Provider>
+    <AppProvider>
```

Comment:
Does the AppProvider need to wrap the entire app?

Consider moving it down to only the level on it is used. (I.e. `<Crops />`)

Reply:
I was using GameContext before, but I was thinking that it was better to not mix different states (application stuff and game stuff).

I put it at App level because maybe can be useful to keep other things :)

I will refactor it as CropsContext!